public/js/index.js

```diff
@@ -1,5 +1,32 @@
-function displayMessage() {
```

Comment:
@arreche I'll take a look at it tonight. It's good that you ask for help, but you shouldn't provide workarounds, especially not for tests, if they're not necessary. You can just check in your failing tests and ask for help here.
I'll see if I can get them working and we'll go over the code together.